### ISSUE TITLE
reformulator and selector links are invalid

### ISSUE THREAD

**Issue by Avi-avidan** [google/active-qa#31]
hi, loved the repo. failed to download pre-trained models using the links in readme.
[checkpoint of the reformulator](https://storage.cloud.google.com/pretrained_models/translate.ckpt-6156696.zip)
and [checkpoint of the selector](https://storage.cloud.google.com/pretrained_models/selector.zip)
are there new updated links?
thanks in advance, a.

**Comment by SageAgastya** [comment]
True, even I checked but it says that my gmail account doesn't have access to those links to download checkpoints.

**Comment by kuya-kevin** [comment]
Hi @rodrigonogueira4  i was wondering if you would be able to fix the links above regarding the model checkpoints? I'm a CMU student looking to use this model for a term project. Thanks!
